mods/ra/maps/allies-01/map.yaml

```diff
@@ -539,9 +539,12 @@ Rules:
 			GenericVisibility: Enemy, Ally, Neutral
 			GenericStancePrefix: false
 			ShowOwnerRow: false
-	^CivInfantry:
+	^ArmedCivilian:
 		RevealsShroud:
 			Range: 0c0
+		AutoTarget:
```

Comment:
Any reason why you put this on `^ArmedCivlian` and not on the actor rules itself as [#8515(diff)](https://github.com/OpenRA/OpenRA/pull/8515/files#diff-0e19aa72642d1c8ffcf6cc9c47e4471cR36) did?